test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -813,84 +636,8 @@ def test_fail_query_workspace_with_load_id_mass(setup_and_teardown):
     ws.get_object_info3.assert_not_called()
 
 
-def test_query_workspace_with_load_id_mass_assembly(setup_and_teardown):
```

Comment:
For the 5 deleted tests do we have equivalent tests for genomes?

Reply:
only `test_fail_upload_assembly_files_in_parallel` doesn't have a matching genome test. But I think that's ok? since it's only testing the failure cases.

Reply:
Not ideal, but we're still under the old test rules so :+1: